packages/db/index.js

```diff
@@ -56,7 +56,7 @@ async function platformaticDB (app, opts) {
     toLoad
   ])
 
-  if (!app.hasRoute({ url: '/', method: 'GET' })) {
+  if (!app.hasRoute({ url: '/', method: 'GET' }) && (!Array.isArray(toLoad) || toLoad.length === 0)) {
```

Comment:
I don't think you applied my diff correctly, please check again. this change should be in `service/index.js`. 

Do you you know that you can apply diffs with the `patch` unix command?

Reply:
Okay, let me update that.

No, how to do that?

Reply:
So for this to work, I should have created a `.diff` file with the changes you showed in the above comment and then apply it, right?